testing/cloudbuild-templates/cloud-run-anthos-template.cloudbuild.yaml

```diff
@@ -0,0 +1,97 @@
+steps:
+- id: 'Generate UUID'
+  name: 'ubuntu'
+  entrypoint: '/bin/bash'
+  args:
+  - '-c'
+  - |
+    cat /dev/urandom | env LC_CTYPE=C tr -dc 'a-z0-9' | fold -w 15 | head -n 1 > _uuid
```

Comment:
This isn't a UUID, just a random string. Maybe call it something like `_short_id`?